lib/generators/setup_generator.rb

```diff
@@ -121,6 +122,13 @@ def gitignore
     return "Orchid .gitignore file copied to app's root directory"
   end
 
+  def helpers
+    FileUtils.rm("#{@new_app}/app/helpers/application_helper.rb")
+    FileUtils.cp(Dir.glob("#{@this_app}/app/helpers/*_helper.rb"), "#{@new_app}/app/helpers/")
```

Comment:
I suppose I could test this out, but this is only copying the reference helpers, and not searching recursively the copy the actual filled out helpers, right?

Reply:
Correct. As intended. If we copy the actual helpers, then Orchid updates would need to be copied into the main app.

This uses the fancier path searching where it knows to look in engines' helper directories for the include files. Similar to how the `include` directive in the main app's `application.js` will look in the asset directories of engines, gems, vendor/, etc and can grab `orchid.js` from the Gem without copying the file into the main app.

Reply:
Right, that's why I wanted to double check that it was only grabbing the referencing helpers.  Cool deal.  :)